examples/xlnet/xlnet_classification_main.py

```diff
@@ -11,22 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Example of building XLNet language model for classification/regression.
+"""
+
+from typing import Optional, Dict
 
 import argparse
 import importlib
 import logging
 import os
 import random
 import time
-from typing import Optional, Dict
 
 import numpy as np
 import torch
+import torch.nn.functional as F
 import tqdm
 import sentencepiece as spm
+
 import texar as tx
 
-import xlnet
+# pylint: disable=wildcard-import
+
+from utils.classification import *
+from utils.glue import *
```

Comment:
Can we put data loaders and stuff under a separate package? It feels weird to import data loaders from `utils`.